package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime",
-  "version": "0.1.4",
+  "version": "0.1.5",
```

Comment:
What's the bump/release strategy for this library?

Reply:
I think we’re trying to follow semver.  Anytime I make a bug fix/minor change to a component, I increase the patch version in prime and then go to app and update the version of prime that app uses.